Tribler/Test/Core/Modules/RestApi/test_libtorrent_endpoint.py

```diff
@@ -4,7 +4,6 @@
 
 
 class TestLibTorrentSettingsEndpoint(AbstractApiTest):
-
```

Comment:
Related to the formatter probably but I would prefer a newline between a class definition and the definition of its first method.

Reply:
Sorry, but `black` thinks otherwise...